src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -50,7 +50,7 @@ class File extends Constraint
     public $uploadFormSizeErrorMessage = 'The file is too large.';
     public $uploadPartialErrorMessage = 'The file was only partially uploaded.';
     public $uploadNoFileErrorMessage = 'No file was uploaded.';
-    public $uploadNoTmpDirErrorMessage = 'No temporary folder was configured in php.ini.';
+    public $uploadNoTmpDirErrorMessage = 'No temporary folder was configured in php.ini, or the configured folder does not exist.';
```

Comment:
This should stay the same. If we made the change here, this would be a BC break for everyone using their own translations. Instead, we can just change the `target` value in the `validators.en.xml` file.